siac/rentercmd.go

```diff
@@ -491,6 +491,7 @@ func renterfilesuploadcmd(source, path string) {
 		for _, file := range files {
 			fpath, _ := filepath.Rel(source, file)
 			fpath = filepath.Join(path, fpath)
+			fpath = filepath.ToSlash(fpath)
```

Comment:
looking at the source, this seems to do a naive `string.Replace`. I feel like that wouldn't work on escaped path separators, but apparently it's good enough for the stdlib.